pkgs/tools/networking/netassert/default.nix

```
@@ -0,0 +1,40 @@
+{ lib
+, buildGoModule
+, fetchFromGitHub
+}:
+
+buildGoModule rec {
+  pname = "netassert";
+  version = "2.0.1";
+
+  src = fetchFromGitHub {
+    owner = "controlplaneio";
+    repo = "netassert";
+    rev = "v${version}";
+    hash = "sha256-bKfqSyG6YXrkHqja8f9R+49mdwOKM5NJuRrcKj7QDj8=";
+  };
+  vendorHash = "sha256-nDnSJOfEn9KieDwdNeIGFcI4m8rVU+Yaxwa+dKyNSHM=";
+
+  ldflags = [
+    "-s"
+    "-w"
+    "-X main.version=${src.rev}"
+  ];
+
+  postBuild = ''
+    mv $GOPATH/bin/{cli,netassert}
+  '';
+
+  meta = with lib; {
+    homepage = "https://github.com/controlplaneio/netassert";
+    changelog = "https://github.com/controlplaneio/netassert/blob/${src.rev}/CHANGELOG.md";
+    description = "NetAssert is a command line utility to test network connectivity between kubernetes objects";
```

```suggestion
    description = "A command line utility to test network connectivity between kubernetes objects";
```

Short `description` should not be self-referent.